src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build.py

```diff
@@ -88,6 +89,11 @@ def acr_build(cmd,
             except OSError:
                 pass
     else:
+        # NOTE: If docker_file_path is not specified, the default is Dockerfile. It's the same as docker build command.
+        if not docker_file_path:
+            docker_file_path = "Dockerfile"
+            logger.warning("'--file or -f' is not provided. '%s' is used.", docker_file_path)
```

Comment:
Does docker build provide a warning like this?  If not, this should be a logger.info statement rather than logger.warning.

Reply:
Thanks for asking. Docker doesn't provide anything. However it's not obvious to everyone if they are new to docker. Sometimes users may have multiple Dockerfiles and are not aware which one is used. So I think providing a message to user is a better experience.

Reply:
The CLI is intended to be quiet by default. If the user wants additional status messages, they specify --verbose. This should be a logger.info and the defaulting behavior should be called out in the parameter help.